### PR TITLE
[docs] Build release notes to documentation

### PR DESCRIPTION
**Pull request by bmorelli25**
For https://github.com/elastic/apm-agent-ruby/issues/556 & elastic/apm#132.

Builds release notes to the documentation. **This will require a separate PR in the docs repo as well and must be merged in the order outlined in #556**. I've saved the file rename for the second commit so you can look at the first commit and easily see the diff.

Screen Shot 2019-10-08 at 12 25 11 PM
Screen Shot 2019-10-08 at 12 25 20 PM

### CODE REVIEW

**Comment by bmorelli25** [comment]
I'm out until Wednesday. Can one of you take a look at this PR before then? I'd like to get this merged (in the right [order](https://github.com/elastic/apm-agent-ruby/issues/556)) on that day if possible. The longer we wait the more changelog entries I have to keep adding to this :)

**Comment by mikker** [comment]
Do we want to keep a note in `CHANGELOG.md` that the content has moved?

**Comment by bmorelli25** [comment]
That's not a bad idea. Might be useful to keep that up for a little while in case there are old external links that point there

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,481 +1 @@
...
-[[release-notes-0.7.4]]
-==== 0.7.4 - 2018-06-07
-
-Beginning of this document
-
-[float]
-===== Fixed
-
-- Fix overwriting custom logger with Rails'
+Release notes are now available in our documentation at ([elastic.co](https://www.elastic.co/guide/en/apm/agent/ruby/current/release-notes.html))
```
Comment (mikker) [inline CHANGELOG.md]: ❤️

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -1 +1,481 @@
...
+[float]
+===== Added
+
+- Add `ElasticAPM::Sinatra.start` API. {pull}556[#566]
+
+[float]
+===== Changes
+
+- Log 404s from CentralConfig on debug level {pull}553[#553]
+
```
Comment (mikker) [inline CHANGELOG.asciidoc]:
```suggestion

[float]
===== Fixed

- Fix Central Config url path {pull}578[#578]

```

**Comment by mikker** [comment]
Test failures are unrelated (and annoying)

**Pull request merged**